src/foundry/foundry.js/pixi/containers/placeableObjects/token.d.ts

```diff
@@ -563,25 +490,37 @@ declare class Token extends PlaceableObject<Token.Data> {
   protected _onDragLeftMove(event: PIXI.InteractionEvent): void;
 
   /**
-   * A factory method to create a Token instance from an Actor entity.
-   * The Token is not automatically saved to the database, it is up to the caller whether or not they wish to do that.
-   *
-   * @param actor     - The input actor entity
-   * @param tokenData - Additional data, such as x, y, rotation, etc. for the created token
-   * @returns The created Token instance
+   * @deprecated since 0.8.0
    */
-  static fromActor(actor: Actor, tokenData?: DeepPartial<Token.Data>): Promise<Token>;
+  static fromActor(actor: Actor, tokenData?: DeepPartial<Token.Data>): never;
```

Comment:
Is `never` the correct return type for a method that always throws an error?

Reply:
Yes